frontend/javascripts/messages.tsx

```diff
@@ -173,6 +173,9 @@ instead. Only enable this option if you understand its effect. All layers will n
     </span>
   ),
   "tracing.copy_cell_id": "Hit CTRL + I to copy the currently hovered segment id",
+  "tracing.segment_id_out_of_bounds": _.template(
+    "Cannot create a segment id larger than the segment layers maximum value of <%- maxSegmentId %>.",
+  ),
```

Comment:
This message shows the maximum value directly and not in a more readable format like `2^32 -1`. 
Pro:
The direct value has the advantage to show the use of the maximum allowed segment id and therefore the user does not need to calculate the maximum value of `2^32 -1` on their own using a calculator or such. 

Con: 
It looks visually not very good.

Which one do you prefer @philippotto ?